sdk/tests/conformance/resources/gl-object-get-calls.js

```diff
@@ -420,12 +435,11 @@ var validArrayForVertexAttrib = new Array(
     gl.VERTEX_ATTRIB_ARRAY_STRIDE,
     gl.VERTEX_ATTRIB_ARRAY_TYPE,
     gl.VERTEX_ATTRIB_ARRAY_NORMALIZED,
-    gl.VERTEX_ATTRIB_ARRAY_STRIDE,
-    gl.VERTEX_ATTRIB_ARRAY_ENABLED,
```

Comment:
Why were the invalid argument tests for these two enums removed? This change does not look good.